src/renderer/utils/jsonPath.ts

```diff
@@ -38,3 +73,35 @@ function convertToIndexNotation(key: string, firstItem = false) {
     return `${prefix}${key}`;
   }
 }
+
+function formatJSONValue(value: unknown) {
```

Comment:
So, format should just remove quotes from a `string` value or `Date` values also supported?

btw: `JSON.stringify(value)` works the same as `String(value)` for non-object values except strings. 
but for Date instances its not equal: `String(date) !== JSON.stringify(date) // true`